app/validators/sections/section_validator.py

```diff
@@ -47,34 +45,20 @@ def validate_summary(self):
 
     def validate_section_enabled(self):
         section_enabled = self.section.get("enabled", None)
+        if not isinstance(section_enabled, dict):
+            return
```

Comment:
Is this needed?. It should now always be a dict?.

Reply:
Now only testing if `section_enabled` is Falsy